testsuite/pytests/test_helper_functions.py

```diff
@@ -25,6 +25,13 @@
 
 class TestHelperFunctions(unittest.TestCase):
 
+    def test_get_help_fname(self):
+        help_text = nest.hl_api.load_help('ac_generator')
+        if help_text:
+            self.assertTrue(isinstance(help_text, str))
+        else:
+            self.assetsIfNone(help_text)
```

Comment:
What's this function? It looks autocorrect-suspiciously much like `assertIsNone` ;p

Reply:
Unit tests are usually deterministic and based on a-priori knowledge, how come you need to check what `help_text` is? Shouldn't we find a case that is always str, and one that is always `None` so that we can confirm both cases?

Reply:
A general comment: Since we now use PyTest, please use plain `assert`s instead of `self.assertXYZ`.